resources/views/layout.blade.php

```diff
@@ -55,43 +55,69 @@
+                    <li class="dropdown dropdown-large">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
+                        <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
+                            <li class="col-sm-12">
                                 <ul>
+                                    <li class="dropdown-header">Membership</li>
+                                    <li>{!! link_to_route("visiting.landing", "Visit/Transfer") !!}</li>
+                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
+                                    <li class="divider"></li>
                                     <li class="dropdown-header">Third-Party Services</li>
                                     <li>{!! link_to_route("teamspeak.new", "TS Registration") !!}</li>
                                     <li>{!! link_to_route("slack.new", "Slack Registration") !!}</li>
-                                    <li class="divider"></li>
-                                    <li class="dropdown-header">Community Features</li>
-                                    <li>{!! link_to_route("mship.notification.list", "View Notifications") !!}</li>
-                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
-                                    <li>{!! link_to_route("mship.feedback.new", "Controller/Pilot Feedback") !!}</li>
                                 </ul>
                             </li>
                         </ul>
-
                     </li>
                 </ul>
 
-                @if(Auth::check())
-                    <ul class="nav navbar-nav navcustom">
+                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                    {!! Form::open(['route' => 'logout', 'id' => 'logout-form']) !!}
+                    <ul class="nav navbar-nav navcustom navbar-right">
                         <li class="dropdown dropdown-large">
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Account <b class="caret"></b></a>
-                            <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
-                                <li class="col-sm-6">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{Auth::user()->name.' (' .Auth::user()->id.')'}} <b class="caret"></b></a>
+                            <ul class="dropdown-menu dropdown-menu-logout dropdown-menu-large row mainmenu_dropdown">
+                                <li class="col-sm-12">
                                     <ul>
+                                        <li class="dropdown-header">Details</li>
+                                        <li><a>{{ $_account->status_string }} {{ !is_null($_account->primary_state) ? $_account->primary_state->name : 'unknown state' }} Member</a></li>
```

Comment:
Not all pages include the $_account variable when the user is authenticated, so this should be Auth::user()